redis_api/api/services.py

```diff
@@ -0,0 +1,24 @@
+import logging
+import os
+from rejson import Client
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.DEBUG)
+
+class RedisClient:
+    def __init__(self):
+        logger.debug(os.environ.get("REDIS_HOST"))
+        self.client = Client(host=os.environ.get("REDIS_HOST", "localhost"), decode_responses=True)
+
+    def getElectionShape(self):
+        return self.getAnyShape('elections')
+
+    def getCandidateShape(self):
+        return self.getAnyShape('candidates')
+
+    def getAnyShape(self, index):
```

Comment:
I like this, I think for flexibility purposes we can just call getAnyShape('key') through .routes. That way, if we wanted to add new endpoints, we wouldn't have to define a bunch of new functions within the RedisClient class.

Reply:
Done